elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -107,6 +107,33 @@ default String getJsonApiPathSpec() {
         return "/api/v1/*";
     }
 
+    /**
+     * API root path specification for the GraphQL endpoint. Namely, this is the root uri for GraphQL.
+     *
+     * @return Default: /graphql/api/v1
+     */
+    default String getGraphQLApiPathSepc() {
+        return "/graphql/api/v1";
```

Comment:
`/v1`? I thought the point of graphql was that it's not versioned?

Reply:
Yeah, we can probably drop the versioning by default if we want. I'm happy with that.

Reply:
I think all APIs should be versioned - even GraphQL.